tools/shared/src/main/scala/org/scalajs/core/tools/linker/backend/emitter/FunctionEmitter.scala

```diff
@@ -2284,8 +2313,10 @@ private[emitter] class FunctionEmitter(jsGen: JSGen) {
         // Atomic expressions
 
         case VarRef(name) =>
-          assert(env.isLocalVar(name), name.name)
-          js.VarRef(transformLocalVarIdent(name))
+          if (env.isLocalVar(name))
+            js.VarRef(transformLocalVarIdent(name))
+          else
+            envField("cc", name.name, name.originalName)
```

Comment:
We should reconsider the name `envField`: `cc` and `superClass` are both fair usages of the `envField` "infrastructure" but blatant abuses of the (implied) semantics. We should name it something like `internalVariable` (but shorter).

This should probably happen later, not in this PR.

Reply:
True. Also I think `b` has the same kind of usage.